src/ClueGaudiAlgorithmWrapper.cpp

```diff
@@ -42,10 +42,72 @@ ClueGaudiAlgorithmWrapper::ClueGaudiAlgorithmWrapper(const std::string& name, IS
 
 StatusCode ClueGaudiAlgorithmWrapper::initialize() {
 
+  bool clue_verbose = false;
+  if (msgLevel(MSG::INFO) || msgLevel(MSG::DEBUG)){
+    clue_verbose = true;
+  }
+
+  auto start = std::chrono::high_resolution_clock::now();
+  clueAlgoBarrel_ = CLICdetBarrelCLUEAlgo(dc, rhoc, outlierDeltaFactor, clue_verbose);
+  auto finish = std::chrono::high_resolution_clock::now();
+  std::chrono::duration<double> elapsed = finish - start;
+  info() << "ClueGaudiAlgorithmWrapper: Set up time (Barrel): " << elapsed.count() * 1000 << " ms" << endmsg;
```

Comment:
```suggestion
  info() << "ClueGaudiAlgorithmWrapper: Set up time (Barrel): " <<  std::chrono::duration_cast<std::chrono::milliseconds>(elapsed) << endmsg;
```
I'm not sure if it works but `std::cout << std::chrono::duration_cast<std::chrono::milliseconds>(elapsed) << std::endl;` and then you don't have to introduce the 1000 factor or the suffix (it's fine as it is but this is another way, also the cast can be done where you define `elapsed`